spec/models/user_spec.rb

```diff
@@ -24,6 +24,11 @@
     it { is_expected.to validate_presence_of(:username) }
     it { is_expected.to validate_uniqueness_of(:username) }
     it { is_expected.to validate_presence_of(:role) }
+    context 'Validate email' do
+      it { should allow_value('user@example.com').for(:email) }
```

Comment:
We are using `is_expected.to` or `expect` syntax in our specs. Plz change to that for consistency.